src/lobby.cpp

```diff
@@ -283,7 +283,10 @@ QString Lobby::get_chatlog()
 
 int Lobby::get_selected_server()
 {
-  return ui_server_list->currentItem()->text(0).toInt();
+  if (auto item = ui_server_list->currentItem()) {
+    return item->text(0).toInt();
```

Comment:
What is the `0` argument? Looks to be an extraneous argument.

Reply:
No idea, it probably is extraneous for us. Docs say it's the column: https://doc.qt.io/qt-5/qtreewidgetitem.html#text

Reply:
Oh, it's a tree widget, not a list widget. Intuitive AO2 code as usual.